lpm/src/prefix/with_ports.rs

```diff
@@ -450,3 +450,434 @@ impl Display for PrefixWithOptionalPorts {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
```

Comment:
Ok, so now we have some tests, but still missing the basic tests for overlap.

Reply:
I added commit `test(config): Add tests for VpcExpose validation with port ranges`, I think it should address your concern (except for fuzzing).